taxi/tests/test_models.py

```diff
@@ -0,0 +1,55 @@
+from django.contrib.auth import get_user_model
+from django.test import TestCase
+
+from taxi.models import Manufacturer, Car
+
+
+class ModelTests(TestCase):
+    @classmethod
+    def setUpTestData(cls):
```

Comment:
It's recommended to use `setUp` instead of `setUpTestData` for creating test data. `setUp` runs before each test method, so it's better for isolation between tests and it helps to avoid unexpected side effects from other tests. But if you're sure that the test data won't be modified during the tests, you can use `setUpTestData` for optimization. It's a trade-off between isolation and performance.